spec/views/users/index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe 'users/index', type: :system do
+  it 'displays users correctly' do
+    # Create some sample users
+    User.create(name: 'John Doe', email: 'john@example.com', password: 'password')
+    User.create(name: 'Jane Smith', email: 'jane@example.com', password: 'password')
```

Comment:
- [ ] Kindly make sure you are using the correct attribute on the instance of your elements. As you may see in your schema, there are no such attributes on user, I mean `email`, `password` rather you have `photo`, `bio` ... Please make sure you fix that.
That's why the test is failing:
![image](https://github.com/joyapisi/blog_app/assets/97100091/393f9206-1f9c-4e61-a696-7497db850825)